src/app/audiologist-summary/audiologist-summary.component.ts

```diff
@@ -206,4 +208,15 @@ export class AudiologistSummaryComponent implements OnInit {
     return list;
   }
 
+  public loadAppointment(app: Appointment) {
+    let appointment = new Appointment(app);
+    this.patientID = appointment.patientid.toString();
+    this.ts = appointment.ts_type;
+    this.tfiVars = appointment.createTfiMap();
+    this.testRadioVars = appointment.createTestSeverityMap();
+    this.testCheckBoxVars = appointment.createTestConfigurationMap();
```

Comment:
I would probably rename 
this.testRadioVars = appointment.createTestSeverityMap();
this.testCheckBoxVars = appointment.createTestConfigurationMap();
to maybe this.testSeverityVars and this.testConfigVars. Not really sure if thats an issue.